packages/core/src/internal/clients/cache/AbstractCacheClient.ts

```diff
@@ -1148,6 +1150,49 @@ export abstract class AbstractCacheClient implements ICacheClient {
     );
   }
 
+  /**
+   * Fetch length (number of items) of sorted set
+   * @param {string} cacheName - The cache containing the sorted set.
+   * @param {string} sortedSetName - The sorted set name.
+   * @returns {Promise<CacheSortedSetLength.Response>}
+   * {@link CacheSortedSetLength.Hit} containing the length if the sorted set exists.
+   * {@link CacheSortedSetLength.Miss} if the sorted set does not exist.
+   * {@link CacheSortedSetLength.Error} on failure.
+   */
+  public async sortedSetLength(
+    cacheName: string,
+    sortedSetName: string
+  ): Promise<CacheSortedSetLength.Response> {
+    const client = this.getNextDataClient();
+    return await client.sortedSetLength(cacheName, sortedSetName);
+  }
+
+  /**
+   * Fetch length (number of items) of sorted set within the provided score range
+   * @param {string} cacheName - The cache containing the sorted set.
+   * @param {string} sortedSetName - The sorted set name.
+   * @param {number} minScore - The lower bound on the score range to search in.
+   * @param {number} maxScore - The upper bound on the score range to search in.
+   * @returns {Promise<CacheSortedSetLengthByScore.Response>}
+   * {@link CacheSortedSetLengthByScore.Hit} containing the length if the sorted set exists.
+   * {@link CacheSortedSetLengthByScore.Miss} if the sorted set does not exist.
+   * {@link CacheSortedSetLengthByScore.Error} on failure.
+   */
+  public async sortedSetLengthByScore(
+    cacheName: string,
+    sortedSetName: string,
+    minScore?: number,
+    maxScore?: number
```

Comment:
For these arguments, I recommend using a single optional object. See for example `listFetch` (or the `sortedSetFetch` methods). In JavaScript there is no way for a caller to omit the `minScore` and pass the `maxScore`; they would have to pass `null` for `minScore` explicitly.

To get around this, the standard is to pass an object, which can take optional properties. The caller can then just omit the optional properties.